setup.py

```diff
@@ -105,6 +105,7 @@ def txt_read(*paths):
         "GeoNodes": ["cartopy[plotting]", ],  # required for visualization of GeoArray and GeoSurface nodes
         "BAG": ["hydroffice.bag>=0.2.10", ],  # required by BAG plugin
         "OpenDAP": ["pydap", ],  # required by OpenDAP plugin
+        "ADIOS": ["adios>=1.9.1b19", ],  # required by ADIOS plugin
```

Comment:
but the thing is: the extra is behaving exactly as it should: is there an adios c library, it can install the extra, otherwise not. independent of the operating system. the python wrapper is not shipping external dependencies (one of the reasons conda was created.)

that's exactly the same behaviour as trying to install wxPython via pip with a missing gstreamer c-library dependencies.